### ISSUE TITLE
Create new `StakingService` method that returns all validators as `ValueView`s

### ISSUE THREAD

**Issue by jessepinho** [penumbra-zone/web#612]

**Comment by hdevalence** [comment]
I think this should be in the view service, if we're expecting it to return info about the user's delegation tokens -- which the chain doesn't have.

How about `rpc DelegationBalances` with a request parameter that has an `all_active_validators` field?

**Comment by jessepinho** [comment]
Thanks @hdevalence — I'll add this to the issue description. `all_active_validators` would determine whether to filter for just active ones, you mean?

**Comment by jessepinho** [comment]
cc @grod220 — this is the second part of moving some staking work into Rust/the extension

**Comment by hdevalence** [comment]
> Thanks @hdevalence — I'll add this to the issue description. `all_active_validators` would determine whether to filter for just active ones, you mean?

I was thinking it would be telling the view service whether to include zero balances at all, the confusion is maybe a sign my name is bad.

- default behavior: like `BalancesRequest` but only returns delegation tokens the user has
- with flag set to `true`: includes one entry for each validator in active set, regardless of balance

**Comment by jessepinho** [comment]
@hdevalence Gotcha. But there wouldn't be a scenario where we'd only want the validators that the user has staked though, right? Wouldn't we always want to show all validators? 

Although, maybe you're thinking of frontends other than ours who might want to arrange that view differently. Anyway, it can't hurt to include that functionality.

**Comment by hdevalence** [comment]
Yeah, exactly. My thought was that from the extension's point of view, the implementation involves querying the user's delegation tokens and then merging that with the validator list. So there's not much extra involved in having a flag to skip the merging, and maybe that's useful for a different use case.
